src/App.js

```diff
@@ -54,8 +81,14 @@ function App() {
                     isNewUser={isNewUser}
                     onLogIn={onPositiveLogIn}
                 />
-                <AppProvider value={{}}>
-                    <Views />
+                <AppProvider>
```

Comment:
no no :p wrap AppProvider into isLoading state and once its loaded pass request result data into context